init.lua

```diff
@@ -42,9 +42,9 @@ local function needs_restart(job_id)
     return false
 end
 
-local function re_run_failed_jobs(run_id)
+local function re_run_failed_jobs(run_id, full_repo)
```

Comment:
Logically it would be better to swap these params:

```
local function re_run_failed_jobs(full_repo, run_id)
    ...
```

Reply:
done